pkg/configuration/configuration.go

```diff
@@ -183,89 +171,14 @@ func validateParamsAPIKey(lapiKey string) error {
 //	tchar = "!" / "#" / "$" / "%" / "&" / "'" / "*" / "+" / "-" / "." /
 //	        "^" / "_" / "`" / "|" / "~" / DIGIT / ALPHA
 //	token = 1*tchar
-func validHeaderFieldByte(b byte) bool {
```

Comment:
This function was in the standard go library of net.  
I guess this form was more perfomant somehow